apollo-integration/src/test/java/com/apollographql/apollo/IntegrationTest.java

```diff
@@ -191,14 +191,12 @@
     latch.awaitOrThrowWithTimeout(TIME_OUT_SECONDS, TimeUnit.SECONDS);
   }
 
-  @Test public void dataEmpty() throws Exception {
+  @Test(expected = ApolloException.class) public void dataEmpty() throws Exception {
```

Comment:
@sav007 I changed this to be an exception. This should never happen per the GQL spec, so seems "exceptional".